apps/condo/domains/miniapp/schema/B2BAppNewsSharingConfig.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Generated by `createschema miniapp.B2BAppNewsSharingConfig 'publishUrl:Text; previewUrl:Text; getRecipientsUrl:Text;'`
+ */
+
+const { Url, Text } = require('@keystonejs/fields')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/miniapp/access/B2BApp')
+
+
+/**
+ * News Sharing B2BApp
+ * 
+ * News Sharing B2BApp allow b2b users to share their NewsItem to external source like Telegram or Whatsapp from /news page
+ *
+ *                         [ whatsapp-sharing-miniapp ] -> [ whatsapp ]
+ * [ condo /news page ] ->              ...                    ...
+ *                         [ telegram-sharing-miniapp ] -> [ telegram ]
+ *
+ * To create miniapp that can be embedded to /news page developer should provide API and information defined here
+ * 
+ * @type {GQLListSchema}
+ */
+const B2BAppNewsSharingConfig = new GQLListSchema('B2BAppNewsSharingConfig', {
+
+    schemaDoc: 'News Sharing B2BApp allow b2b users to share their NewsItem to external source (like Telegram) from /news page',
+    fields: {
+        name: {
+            schemaDoc: 'Short and simple name of the external source. For example: Telegram',
+            type: Text,
+            isRequired: true,
+        },
+
+        publishUrl: {
+            schemaDoc: 'URL that implements publishing NewsItem method',
+            type: Url,
+            isRequired: true,
+        },
+
+        previewUrl: {
+            schemaDoc: 'URL that returns HTML preview NewsItem',
+            type: Url,
+            isRequired: true,
+        },
+
+        getRecipientsUrl: {
+            schemaDoc: 'URL that implements getRecipients function',
+            type: Url,
+            isRequired: true,
+        },
+    },
+    plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
+    access: {
+        read: access.canReadB2BApps,
```

Comment:
It's not obvious to share access here, since we have direct access with its own config as replacement for support / admin users. So I will not figure out that 'canReadB2BApps` / `canManageB2BApps` checkboxes will provide me access to N more models...

Reply:
Thanks for the question

While I agree on the fact that it is not obvious that giving access to B2BApp model also gives access to B2BApp.newsSharingConfig model, I cannot think of any situation when you may need to restrict access to **Config model